docs/cspell.json

```diff
@@ -489,6 +490,7 @@
     "loginuid",
     "loginwithmsft",
     "logrus",
+    "longterm",
```

Comment:
I think we should change the placeholder value where we use this to say `long-term` instead. Otherwise, we'll ignore uses of "longterm" in other locations.